test/master-playlist-controller.test.js

```diff
@@ -2033,36 +2033,6 @@ QUnit.test(
 
     mediaChanges.length = 0;
 
-    endList = false;
-    currentTime = 100;
-    currentPlaylistBandwidth = 1000;
-    nextPlaylistBandwidth = 1001;
-    buffered = [];
-    this.masterPlaylistController.mainSegmentLoader_.trigger('bandwidthupdate');
-    assert.equal(
-      mediaChanges.length,
-      1,
-      'changes live media when no buffer and higher bandwidth playlist'
-    );
-    buffered = [[0, 100], [100, 109]];
-    this.masterPlaylistController.mainSegmentLoader_.trigger('bandwidthupdate');
-    assert.equal(
-      mediaChanges.length,
-      2,
-      'changes live media when insufficient forward buffer and higher ' +
-               'bandwidth playlist'
-    );
-    buffered = [[0, 100], [100, 130]];
-    this.masterPlaylistController.mainSegmentLoader_.trigger('bandwidthupdate');
-    assert.equal(
-      mediaChanges.length,
-      3,
-      'changes live media when sufficient forward buffer and higher ' +
-               'bandwidth playlist'
-    );
-
-    mediaChanges.length = 0;
-
```

Comment:
Do we want to remove these tests?

Reply:
These only tested that live content always switches as far as I could tell. We might have to add one more back in depending on changes that I have to make in shouldSwitchToMedia.